extensions/amp-story-auto-analytics/validator-amp-story-auto-analytics.protoascii

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-story-auto-analytics
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-story-auto-analytics"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-story-auto-analytics>
+  html_format: AMP
+  tag_name: "AMP-STORY-AUTO-ANALYTICS"
+  requires_extension: "amp-story-auto-analytics"
+  attrs: {
```

Comment:
This one should be mandatory.

Can we validate it using a regex like `UA-[0-9]+-[0-9]+`?

Reply:
Added mandatory, but as we discussed regex would be complex due to the number of different types of tags so we're not enforcing a regex on the value (+ future compatibility)

Reply:
Could you link the discussion?

Reply:
And regarding `future compatibility`. It's easier to be more restrictive and ease those than to be less restrictive and try to tighten it.